atlasdb-client/src/main/java/com/palantir/atlasdb/logging/LoggingArgs.java

```diff
@@ -46,7 +46,6 @@
 
     private static final String PLACEHOLDER_TABLE_NAME = "{table}";
 
-    @VisibleForTesting
     public static final TableReference PLACEHOLDER_TABLE_REFERENCE =
```

Comment:
is this technically only visible for testing? If so, can we just drop a comment here instead?

Reply:
no worries then - I guess the annotation was just bad here!